configs/RISCV_TX2.yaml

```diff
@@ -0,0 +1,140 @@
+---
+# The following resources where utilised to create the config file and naming schemes:
+# https://en.wikichip.org/wiki/cavium/microarchitectures/vulcan
+
```

Comment:
This model was provided as an example RISC-V core for external collaborators so that it can be kept up to date with any changes we make to the config structure. I will make the second change you suggested